integrations/haproxy/documentation.yaml

```diff
@@ -1,27 +1,34 @@
-exporter_type: sidecar
+exporter_type: included
 app_name_short: HAProxy
 app_name: {{app_name_short}}
 app_site_name: HAProxy
 app_site_url: https://www.haproxy.com/
-exporter_name: the HAProxy exporter
-exporter_pkg_name: haproxy_exporter
-exporter_repo_url: https://github.com/prometheus/haproxy_exporter
+exporter_name: the HAProxy promex exporter
+exporter_pkg_name: haproxy_promex
+exporter_repo_url: https://github.com/haproxy/haproxy/blob/master/addons/promex/README
 dashboard_available: true
-minimum_exporter_version: v0.13.0
+minimum_exporter_version: "2.4"
 multiple_dashboards: false
 dashboard_display_name: {{app_name_short}} Prometheus Overview
-config_mods: |
+additional_prereq_info: |
+  {{app_name_short}} exposes Prometheus-format metrics only when it is 
+  [built with the service enabled](https://github.com/haproxy/haproxy/blob/master/addons/promex/README) and an appropriate `frontend` is included in the configuration.
```

Comment:
[text](link){:class="external"}, so it's like this:

when it is [built with ...](https://github.com/.../README){:class="external"} and ...